workflow/react-project/src/components/Modal/Modal.js

```diff
@@ -3,12 +3,13 @@ import { faTimes } from '@fortawesome/free-solid-svg-icons';
 import { faStar } from '@fortawesome/free-regular-svg-icons';
 import { closeModal } from '../../Redux/Actions/modalAction';
 import { useSelector, useDispatch } from 'react-redux';
+import classNames from 'classnames';
 
 
-function Modal({ randomCoctail }) {
+function Modal({ randomCocktail }) {
 
-	const title = randomCoctail.title;
-	const { contentTitle, imgPath, instruction, ingrQnty } = randomCoctail.cocktail;
+	const title = randomCocktail.title;
+	const { contentTitle, imgPath, instruction, ingredirntQuantity } = randomCocktail.cocktail;
```

Comment:
can be combined into: 
`
const { cocktail: {contentTitle, imgPath, instruction, ingredirntQuantity}, title } = randomCocktail;
`